src/object.d

```diff
@@ -1538,7 +1538,8 @@ class Throwable : Object
         string toString() const;
     }
 
-    string      msg;    /// A message describing the error.
+    string      msg;    // A message describing the error
```

Comment:
This is better left documented as it were for now, thus making the adding of `message()` independent of any undocumenting/deprecation mechanism etc.